exercises/C-comparison-operators/exercise.js

```diff
@@ -7,14 +7,14 @@
 
 var studentCount = 16;
 var mentorCount = 9;
-var moreStudentsThanMentors; // finish this statement
+var moreStudentsThanMentors = studentCount > mentorCount; // finish this statement
 
 var roomMaxCapacity = 25;
-var enoughSpaceInRoom; // finish this statement
+var enoughSpaceInRoom = roomMaxCapacity === studentCount + mentorCount // finish this statement
```

Comment:
I suggest you change this line to:
var enoughSpaceInRoom = studentCount + mentorCount <= roomMaxCapacity;
Although your code is working fine with the test, if number of students and mentors would be less than 25, the answer to the question of " Is there enough space in the room for all students and mentors? " would return false with your code